crates/bevy_tasks/src/iter/adapters.rs

```diff
@@ -194,16 +194,16 @@ pub struct Cloned<P> {
     pub(crate) iter: P,
 }
 
-impl<'a, B, P, T> ParallelIterator<std::iter::Cloned<B>> for Cloned<P>
+impl<'a, B, P, T> ParallelIterator<std::iter::Copied<B>> for Cloned<P>
 where
     B: Iterator<Item = &'a T> + Send,
     P: ParallelIterator<B, Item = &'a T>,
     T: 'a + Copy,
```

Comment:
was this bound meant to be `Clone` by any chance?

Reply:
Yeah, that seems suspicious.

Reply:
Probably meant to be Clone. But lets revert this for now. I don't want to conflate functionality changes / bugfixes with style changes.

Reply:
(feel free to open a new pr or issue to discuss and address this)